docs/fides/docs/quickstart/docker.md

```diff
@@ -1,12 +1,12 @@
-# Running Fidesctl in Docker
+# Running Fidesctl with Docker Compose
```

Comment:
On the title selection, can you explain why you decided "with docker compose" vs "in docker"?

Reply:
Providing the information that Docker Compose is a requirement early on might help prevent a reader from assuming they only needed a docker install, and skipping down to the code samples/content without reading further.